static/src/javascripts/boot.js

```diff
@@ -75,11 +75,11 @@ const go = () => {
                                                     document.readyState ===
                                                     'complete'
                                                 ) {
-                                                    capturePerfTimings();
+                                                    capturePerfTimings.capturePerfTiming();
```

Comment:
Can we destructure the object on import? Helps with tree shaking, also looks nice in the code:
```
const { capturePerfTimings } = require('lib/capture-perf-timings');

// later
capturePerfTimings();
```